src/test/java/com/lazerycode/jmeter/configuration/JMeterArgumentsArrayTest.java

```diff
@@ -388,7 +388,7 @@ public void resultsFileHasCorrectName() throws Exception {
 		assertThat(testArgs.getResultsLogFileName(),
 				is(not(equalTo(null))));
 		assertThat(UtilityFunctions.humanReadableCommandLineOutput(testArgs.buildArgumentsArray()),
-				is(equalTo("-d target/jmeter/" + " -l " + "temp/test.jtl" + " -n -t " + testFile.getAbsolutePath())));
+				is(equalTo("-d target/jmeter/" + " -l " + "temp" + File.separator + "test.jtl" + " -n -t " + testFile.getAbsolutePath())));
 	}
```

Comment:
it looks like we have left `target/jmeter/` with unix style file paths which is giving us a mix of windows and unix style  which looks wrong.  Is this a bug we have previously missed?